tests/acceptance/run.sh

```diff
@@ -96,6 +96,9 @@ if [ -n "${EXPECTED_FAILURES_FILE}" ]; then
       continue
     fi
 
+    if [[ "${LINE}" =~ -\ \[(.*?)] ]]; then
+      LINE="${BASH_REMATCH[1]}"
+    fi
```

Comment:
```suggestion
    if [[ "${LINE}" =~ -\ \[(.*?)] ]]; then
      LINE="${BASH_REMATCH[1]}"
    else
      continue
    fi
```
I think that you will need to `continue` so that lines that did not match the `- [xxx]` format will be skipped. Otherwise the code below will try to match the "random line of text" to a failed scenario.